internal/config/config.go

```diff
@@ -207,11 +207,11 @@ func (tc *TestConfig) Compare(results *TestResults) (*CompareResult, error) {
 		case Skip:
 			switch testRes.Status {
 			case Fail:
-				compareResult.UnexpectedFail[test] = testResOutput
+				compareResult.ExpectedFail[test] = testResOutput
```

Comment:
If we expected (`expectedRes`) test to be skipped (`Skip`) and it actually (`testRes.Status`) failed (`Fail`) instead, that's unexpected by definition. That change and the changes below make CI always green by effectively ignoring all test results.

That's one way to fix everything, for sure, but not the best one 😆

Reply:
This was an unintended commit! I will revert now to at least show that initial bug.

Reply:
> Please send a separate PR with a code fix

So I did that, and now a bunch of configurations fail

Reply:
Of course we have to fix the config files.

Reply:
Once we merge #751 the configuration failures will be fixed.